dataLicense_test.go

```diff
@@ -0,0 +1,18 @@
+package thingfulx
+
+import (
+	"fmt"
+	// "github.com/davecgh/go-spew/spew"
+	"encoding/json"
+	"testing"
+)
+
+func TestLicense(t *testing.T) {
+	j, _ := json.Marshal(CC_BY_NC_V3)
+	fmt.Println(string(j))
```

Comment:
I know you did this little test case because we've (or at least I've) always talked at tedious length about the benefit of test suites, and that's good. 
But this type of test is only useful for a human to look at and verify correctness; whereas the whole point of automated testing is to allow the computer to verify things for you.

One way to do this would be to change this test into something the machine is able to verify, i.e. by asserting the serialized value is the string you expect, maybe something like:

```go
j, err := json.Marshal(CC_BY_NC_V3)
assert.Nil(t, ,err)
expected := `{"expected":"json"}`
assert.Equal(t, expected, string(j))
```